src/components/App/App.js

```diff
@@ -2,42 +2,58 @@ import React, {Component} from 'react';
 import './App.css';
 import Form from '../Form/Form';
 import Movies from '../Movies/Movies';
-import movieData from '../../data/movieData';
 import SingleMovie from '../SingleMovie/SingleMovie';
+import getData from '../../apiCalls/api';
 
 class App extends Component {
  constructor() {
   super();
   this.state = {
     allMovies: [],
     singleMovieId: 0,
```

Comment:
I'd argue we don't need a separate piece of state for the ID of the movie - that data is already held in the `singleMovie` object. Try to keep your state as slim as possible!